notebooks/official/generative_ai/text_embedding_api_cloud_next_new_models.ipynb

```diff
@@ -0,0 +1,363 @@
+{
```

Comment:
<ul><li>Change "colab" to "notebook"</li><li>Recommend to follow the notebook template https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb to add Objective and Dataset/Models sections.</li><li>Recommend to provide more information in the the overview.</li><li>Change "Learn more about text embedding api" to "Learn more about text embeddings"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>